ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -277,25 +277,32 @@ final class Authorizer(
       authorized: (ClaimSet.Claims, Req) => Either[StatusRuntimeException, Req]
   ): (Req, StreamObserver[Res]) => Unit = (request, observer) => {
     val serverCallStreamObserver = assertServerCall(observer)
-    authenticatedClaimsFromContext()
-      .fold(
-        ex => {
-          observer.onError(ex)
```

Comment:
An exception thrown from the `call` function is passed down to `onError` and inside there converted to a StatusRuntimeException through `Status.fromThrowable(t)`

Reply:
Why does the `Authorizer` need to know about the exception types and how to handle them? A thrown exception in `call` should be one of the following:
* a `StatusRuntimeException` - we assume this type is orderly created by our self service err code infra and implicitly logged
* a more generic exception type (e.g. `RuntimeException`) - which should also go through the interceptor and wrapped there in an internal.

Reply:
The problem might actually be here: https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/apiserver/error/ErrorInterceptor.scala#L59-L63. There we only wrap `INTERNAL` error codes and treat everything else as the happy path. But you can trigger also `UNKNOWN` by randomly throwing exceptions in handlers.

Reply:
I think we need to handle `UNKNOWN` in that if branch as well

Reply:
Thanks @tudor-da for the review. 
I prepared a separate PR following your suggestion here: https://github.com/digital-asset/daml/pull/15765